SRC/NETWORK/sntp.c

```diff
@@ -86,25 +84,29 @@ enum TcpipErrorCodes {
 };
 
 const char* strPresentation=
-    "SNTP time setter for the TCP/IP UNAPI 1.1\r\n"
-    "by Oduvaldo\r\n"
-	"based on SNTP 1.0 by Konamiman\r\n"
+    "SNTP Time Setter v1.2\r\n"
+    "for the TCP/IP UNAPI 1.1\r\n"

```

Comment:
Better something like this:

SNTP time setter for the TCP/IP UNAPI 1.1
By (your name or nick), Oduvaldo and Konamiman

Reply:
Hi there!
Just change the help string as requested :)

Didn't add my name, because I'm ok with my name only on version control ;)